features/admin.branding.v1/constants/branding-preferences-constants.ts

```diff
@@ -866,6 +870,11 @@ export class BrandingPreferencesConstants {
     };
 
     public static readonly APP_WISE_BRANDING_FEATURE_TAG: string = "branding.applicationLevelBranding";
+
+    /**
+     * This is the feature id for the self service custom layout feature.
+     */
+    public static readonly CUSTOM_PAGE_EDITOR_FEATURE_ID: string = "branding.design.layout.custom";
```

Comment:
now we define all feature ids in feature-gate.v1

https://github.com/wso2/identity-apps/blob/master/features/admin.feature-gate.v1/constants/feature-flag-constants.ts#L33